### ISSUE TITLE
Cannot install Analog with v15 template.

### ISSUE THREAD

**Issue by VentGrey** [analogjs/analog#199]
### Please provide the environment you discovered this bug in.

I used the cli tool. Ran the command `npm create analog@latest` and got an error in the install section.

### Which area/package is the issue in?

create-analog

### Description

The latest Analog installer fails at starting a new project if the v15 template is used. I don't know if this is a duplicate of #188 or a new bug.

### Please provide the exception or error you saw

```true
Ok to proceed? (y) y
✔ Project name: … myapp
✔ Select a template: › Analog
✔ Select a variant: › angular-v15
...
Done. Now run:
  cd myapp
  npm install
  npm run dev
npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: myapp@0.0.0
npm ERR! Found: vite@4.0.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^4.0.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.2.3" from @nrwl/vite@15.3.3
npm ERR! node_modules/@nrwl/vite
npm ERR!   dev @nrwl/vite@"~15.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/ventgrey/.npm/_logs/2023-01-08T03_50_26_978Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ventgrey/.npm/_logs/2023-01-08T03_50_26_978Z-debug-0.log
```
```


### Other information

The complete log of `/home/ventgrey/.npm/_logs/2023-01-08T03_50_26_978Z-debug-0.log` is:

```

```

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [X] No

**Comment by Yberion** [comment]
Hello, you need to use `npm install --legacy-peer-deps`

**Comment by brandonroberts** [comment]
@VentGrey that issue is fixed, just not in the latest release. As a temp workaround, in the vite.config.ts add 

```json
optimizeDependencies : {
  exclude: ['@analogjs/router']
}
```

**Comment by VentGrey** [comment]
Didn't know that :sweat_smile: should I close this then? :thinking:

**Comment by brandonroberts** [comment]
You can leave it open. I'll close it with the next release

**Comment by brandonroberts** [comment]
Fix released in `@analogjs/platform` 0.1.0-beta.8
